src/pushInfo/PushInfo.story.mdx

```diff
@@ -0,0 +1,42 @@
+import { Meta, Story, Preview, Props } from '@storybook/addon-docs/blocks'
+
+import { ProximityIcon } from '../icon/proximityIcon'
+import { PushInfo } from './index'
+
+<Meta title="Widgets|PushInfo" />
+
+# **PushInfo**
+
+### Default
+
+<Preview>
+  <Story name="Default">
+    <PushInfo
+      headline="This is the headline!"
+      content="But we can also give you more explanations here"
+    />
+  </Story>
+</Preview>
+
+### With icon
+
+<Preview>
+  <Story name="With icon">
+    <PushInfo
+      icon={<ProximityIcon title="" />}
+      headline="This is the headline!"
+      content="But we can also give you more explanations here"
+    />
+  </Story>
+</Preview>
+
+## Specifications
+
```

Comment:
Maybe say that we should add some colors to the Icon?

Reply:
I don't think it's necessary, it's part of the icon, not the PushInfo.
I added/adapted some of the specs from the designers.